default-templates/new-account/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{#if name}}{{name}} &ndash; {{/if}}Solid Home</title>
+  <title>Personal Data Space (or POD) {{#if name}} of {{name}} {{/if}}</title>
```

Comment:
These changes are unrelated to this PR.

Reply:
So can't you just remove the index.html then?

Reply:
Rollback this fix